shared-bindings/displayio/__init__.c

```diff
@@ -45,6 +45,23 @@
 //| """
 //|
 
+//| AnyDisplayBus: fourwire.FourWire | i2cdisplaybus.I2cDisplayBus
+//| """Type-checking shorthand for any kind of display bus. Not actually defined in CircuitPython."""
+//|
+//| AnyFramebuffer: (
```

Comment:
I'm pretty sure this should be
```suggestion
//| AnyFramebuffer = (
```
As in... `AnyFramebuffer` is any of these types, not to be **an instance** of them

---
On a quick check in a blank file
```py
foo: int | float
bar: foo  # Variable not allowed in type expression (by pylance extension on VSCode)
```
```py
foo = int | float
bar: foo  # nothing reported
```
---

PS: Same thing applies below

Reply:
thanks! This is a mistake I commonly make.